persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_100_redis/Config.toml

```diff
@@ -0,0 +1,4 @@
+[persist_generate_100.entities.redis]
+connection = "redis://localhost:6379"
+
+
```

Comment:
```suggestion
```

Reply:
This has been fixed in https://github.com/ballerina-platform/persist-tools/pull/333/commits/a47e9c401642d1ee32077e4b83f1420855854ec4